### DISCUSSION TITLE
point_0699

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#251] [Roy]
"20200712T050659_20200712T051957_T44RMQ"
![image](https://user-images.githubusercontent.com/54947195/147514929-34181cc4-3a0f-4d9c-a414-548635b6a4da.png)
![image](https://user-images.githubusercontent.com/54947195/147514977-272fc28e-3b66-40dc-920e-41a45319f40a.png)

"20200526T051649_20200526T052400_T44RMQ"
![image](https://user-images.githubusercontent.com/54947195/147515095-04a28a35-e917-45f4-8ad7-e1072004dac2.png)
![image](https://user-images.githubusercontent.com/54947195/147515167-8c772648-eb65-4fbd-8670-0ae850cbe194.png)

"20190514T050701_20190514T051547_T44RMQ"
![image](https://user-images.githubusercontent.com/54947195/147515223-3af570a6-47fc-463b-b55c-da9d2b724db9.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20200712T050659_20200712T051957_T44RMQ"
![Captura](https://user-images.githubusercontent.com/86502862/152918106-5a74a393-f77a-4b7a-ac07-ad79e8a7662f.JPG)
